src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
Lets extend the error messages... We should also do it with ShouldBe I guess. https://github.com/shouldly/shouldly/blob/master/src/Shouldly.Tests/Strings/ShouldBe/CaseIsInsensitiveScenario.cs#L15